### ISSUE TITLE
[HELP] Managing Multiple session

### ISSUE THREAD

**Issue by sunel** [SerayaEryn/fastify-session#102]
Can we manage multiple session using this library  ??

**Comment by mfpopa** [comment]
I don't think it works. I tried it and it throws `FST_ERR_DEC_ALREADY_PRESENT: The decorator 'sessionStore' has already been added!`.

The plugin creates and decorates the request with 3 things:

```javascript
fastify.decorateRequest('sessionStore', options.store)
fastify.decorateRequest('session', {})
fastify.decorateRequest('destroySession', destroySession)
```

 It does this internally and I don't see any options to change this on demand, so my conclusion is that you cannot manage multiple sessions with this plugin.

**Comment by SerayaEryn** [comment]
It currently not possible to manage multiple sessions with `fastify-session`.

Free free to open a PR if this is still needed.
